apps/finance/contracts/Finance.sol

```diff
@@ -74,8 +73,7 @@ contract Finance is AragonApp, ERC677Receiver {
         mapping (address => bool) hasBudget;
     }
 
-    Vault public vault;
-    EtherToken public etherToken;
+    IVaultConnector public vault;
```

Comment:
Seems a bit odd that `vault` is an `IVaultConnector`; I haven't looked yet, but I guess `Vault` builds on that interface too?

Reply:
Ya, it looks odd, but it's actually cool: `Vault` doesn't implement it, but fallback is doing `delegateFwd` to it. This way final functions can be called without the need of implementing them in `Vault`. It was @izqui 's suggestion, so pinging him here.